### ISSUE TITLE
Seeder/Leecher columns empty

### ISSUE THREAD

**Issue by Tharn** [equeim/tremotesf2#245]
Hi there,

Thanks for making an alternative to TRGUI.

Right now, Seeder and Leecher columns show '0' for most torrents.
It only shows a number when the torrent is being uploaded/downloaded, and after it goes back to idle that number is retained. But for idle seeding torrents the client does not display anything. I think TRGUI behavior is more typical here and would be an improvement.

**Comment by Dustie** [comment]
Yes, I see the same here. I thought something was wrong at either my end or the tracker for a moment there. 
Win client, Debian server.

**Comment by Tharn** [comment]
Thanks, it's an improvement.

Right now I'm never seeing 'Leechers' column populated though, even on active uploads.

Also, the 'Downloading to peers', 'Uploading to peers' columns take up rather a lot of screen real estate, partly because of the long titles.

I would propose to simplify this in a TRGUI way, having just the classic Seeders and Leechers columns. `Seeders: 108 (2)` meaning 108 seeders reported by tracker, downloading from 2 seeders at the moment. And `Leechers: 2 (1)` meaning 2 leechers reported by the tracker, connected to 1 of those for either up- or download.

Would that be sensible?

**Comment by equeim** [comment]
>Right now I'm never seeing 'Leechers' column populated though, even on active uploads.

Oof, I just found a bug that makes leechers always zero. Will fix shortly.

Also, seeders/leechers still can be zero even for active torrents. They are reported from tracker and it may not return these values (I'm not sure but there are multiple protocols for client-tracker communication). Also it possible for torrent not have trackers at all in which case these will be zero.

BTW "Peers that we are downloading from" can also include leechers. We can download from peers that have 100% of data (seeders), as well as peers that don't. Tracker's number of seeders contains only seeders, obviously.

>I would propose to simplify this in a TRGUI way, having just the classic Seeders and Leechers columns. Seeders: 108 (2) meaning 108 seeders reported by tracker, downloading from 2 seeders at the moment. And Leechers: 2 (1) meaning 2 leechers reported by the tracker, connected to 1 of those for either up- or download.

I think it will be confusing considering that it is possible for numbers returned from tracker to be zero while we are actively downloading/seeding. Also, these are just fundamentally different metrics ("number of seeders that tracker thinks are active right now" and "number of peers that we are downloading from right now") and we shouldn't combine them together without clear differentiation.

**Comment by Tharn** [comment]
Are they useful for screen real estate? Look at how compact uTorrent and TRGUI are when it comes to Seeders/Leechers. I appreciate wanting to be fastidious, but at least an option to condense these down would be appreciated.

This is not uncommon:
![Clipboard-1](https://github.com/equeim/tremotesf2/assets/13466852/e5dce5a3-f1df-4241-9e92-220a658c1129)
